contracts/V1/StateChannelManagerEvents.sol

```diff
@@ -19,21 +19,34 @@ interface StateChannelManagerEvents {
     event DisputeUpdated(bytes32 indexed channelId, Dispute dispute);
 
     event DisputeCommited(
-    bytes encodedDispute,
-    uint timestamp
+        bytes encodedDispute,
+        uint timestamp,
+        bytes32 disputeCommitment
```

Comment:
why is disputeCommitment also emitted? The client can just hash the encodedDispute and timestamp

Reply:
convenience. it is redundant